async-nats/src/options.rs

```diff
@@ -519,6 +522,24 @@ impl ConnectOptions {
         self.lame_duck_callback = CallbackArg0::<()>(Box::new(move || Box::pin(cb())));
         self
     }
+
+    /// By default, Client dispatches op's to the Client onto the channel with capacity of 128.
+    /// This option enables overriding it.
+    ///
+    /// # Examples
+    /// ```
+    /// # #[tokio::main]
+    /// # async fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
+    /// async_nats::ConnectOptions::new().send_capacity(256).connect("demo.nats.io").await?;
+    /// # Ok(())
+    /// # }
+
+    /// ```
+    ///
+    pub fn send_capacity(mut self, capacity: usize) -> ConnectOptions {
```

Comment:
lgtm but a bit of Saturday bike shedding 😁 

We aren't exposing send anywhere, you'll run into this blocking on `publish`, `subscribe` etc so we should take a minute with this naming.

 ````suggestion
    pub fn send_capacity(mut self, capacity: usize) -> ConnectOptions {
```

Reply:
I was not sure about naming too.
Thinking about it now - we should have separate option for sender and receiver.
One might want to receive as much as possible and buffer it (this will play well with `slow consumers` and have much smaller send buffer.